### ISSUE TITLE
Java 11 support

### ISSUE THREAD

**Issue by IlyaGazman** [AlmasB/FXGL-MobileApp#3]
Do mobile apps support Java 11? I want to deploy to web and Android how should I configure my project?

**Comment by AlmasB** [comment]
Yes, but only at the proof-of-concept level. See [this demo](https://github.com/AlmasB/FXGLGames/tree/master/Drop), which has everything set up correctly. There are still various issues at the native level, not related to FXGL, so until those are fixed, I would not recommend deployment to mobile.
